sdk/core/Azure.Core/src/Operation{T}.cs

```diff
@@ -53,8 +53,8 @@ public abstract class Operation<T> where T : notnull
         /// <summary>
         /// Periodically calls the server till the long-running operation completes.
         /// </summary>
-        /// <param name="cancellationToken"></param>
-        /// <returns></returns>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> used for the periodical service calls.</param>
+        /// <returns>The last HTTP response received from the server.</returns>
```

Comment:
In parallel to the Form Recognizer changes, adding some missing info in `Operation{T}`.